oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/config/ProviderSettings.java

```diff
@@ -146,10 +151,10 @@ public ProviderSettings tokenRevocationEndpoint(String tokenRevocationEndpoint)
 
 	protected static Map<String, Object> defaultSettings() {
 		Map<String, Object> settings = new HashMap<>();
-		settings.put(AUTHORIZATION_ENDPOINT, "/oauth2/authorize");
-		settings.put(TOKEN_ENDPOINT, "/oauth2/token");
-		settings.put(JWK_SET_ENDPOINT, "/oauth2/jwks");
-		settings.put(TOKEN_REVOCATION_ENDPOINT, "/oauth2/revoke");
+		settings.put(AUTHORIZATION_ENDPOINT, OAuth2AuthorizationEndpointFilter.DEFAULT_AUTHORIZATION_ENDPOINT_URI);
```

Comment:
Please revert the changes in `ProviderSettings` as it should not depend on `OAuth2AuthorizationEndpointFilter` or the other `Filter`'s

Reply:
Regarding this change, default provider settings must be initialized with filter's default value. Don't you think ? Maybe default endpoint URI in filters become useless.

Reply:
I rollbacked this change